crates/rooch-types/src/multichain_id.rs

```diff
@@ -0,0 +1,203 @@
+// Copyright (c) RoochNetwork
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::{bail, format_err, Result};
+#[cfg(any(test, feature = "fuzzing"))]
+use proptest_derive::Arbitrary;
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+use std::fmt;
+use std::fmt::{Display, Formatter};
+use std::str::FromStr;
+
+use crate::chain_id::ChainID;
+
+pub const BITCOIN: u64 = 0;
+pub const ETHER: u64 = 60;
+pub const SUI: u64 = 784;
+pub const NOSTR: u64 = 1237;
+pub const ROOCH: u64 = 20230103; // Dev
```

Comment:
Use main chain_id as multichain_id. We can also use another id here, using the same value as chain_id just for easy to remember.

Reply:
I gave it to a slip-0044 placeholder. Now it's 20230101.